Source/PauseMenu/TabMissionSelectItem.cs

```diff
@@ -178,47 +280,59 @@ public override void Draw()
             {
                 drawTexture = false;
                 Sprite sprite = Heists[Index].Logo.Sprite;
-                sprite.Position = new Point((int)res.Width - SafeSize.X - 512, SafeSize.Y);
-                sprite.Size = new Size(512, 256);
+                sprite.Position = missionPoint.AddPoints(new Point(logoPaddingLeft, 0));
+                sprite.Size = logoSize;
                 sprite.Color = Color.FromArgb(blackAlpha, 255, 255, 255);
                 sprite.Draw();
             }
             else
             {
                 drawTexture = false;
             }
+            
+            // padding around texture background
+            if(DynamicLogoRatio)
+            {
+                ResRectangle.Draw(missionPoint, new Size(logoPaddingLeft, logoArea.Height), Color.FromArgb(blackAlpha, Color.Black));
+                ResRectangle.Draw(missionPoint.AddPoints(new Point(logoArea.Width - logoPaddingRight, 0)), new Size(logoPaddingRight, logoArea.Height), Color.FromArgb(blackAlpha, Color.Black));
+            }
 
-            ResRectangle.Draw(new Point((int)res.Width - SafeSize.X - 512, SafeSize.Y + 256), new Size(512, 40), Color.FromArgb(fullAlpha, Color.Black));
-            ResText.Draw(Heists[Index].Name, new Point((int)res.Width - SafeSize.X - 4, SafeSize.Y + 260), 0.5f, Color.FromArgb(fullAlpha, Color.White), Common.EFont.HouseScript, ResText.Alignment.Right, false, false, Size.Empty);
+            // description text and background rendering
+            ResRectangle.Draw(missionPoint.AddPoints(new Point(0, logoArea.Height)), new Size(logoArea.Width, 40), Color.FromArgb(fullAlpha, Color.Black));
+            ResText.Draw(Heists[Index].Name, missionPoint.AddPoints(new Point(logoArea.Width - 4, logoArea.Height + 4)), 0.5f, Color.FromArgb(fullAlpha, Color.White), Common.EFont.HouseScript, ResText.Alignment.Right, false, false, Size.Empty);
```

Comment:
Long names will look better if the background is resized and the text wrapped, like the description.

Reply:
Resolved by d2ac036

Reply:
![271590_20220316012630_1](https://user-images.githubusercontent.com/17376331/158548098-cd7b187a-3e6c-4e70-99f8-8866c37caa59.png)